colossalai/pipeline/schedule/zero_bubble_pp.py

```diff
@@ -124,6 +129,7 @@ def assert_buffer_empty(self):
         assert len(self.recv_backward_buffer[1]) == 0
         assert len(self.local_send_forward_buffer) == 0
         assert len(self.local_send_backward_buffer) == 0
+        # assert len(self.send_handles) == 0
```

Comment:
why

Reply:
> why

Removed in https://github.com/hpcaitech/ColossalAI/pull/6083/commits/fafe049b83bad3a6aa6e3a31c68b38ac63167b53;
Cause I travel throughout it instead of pop it.
![Uploading img_v3_02g4_686f14b2-9b4b-47e3-bef2-936a8579beag.jpg…]()